test/OWNERS

```diff
@@ -0,0 +1,7 @@
+# The OWNERS file is used by prow to automatically merge approved PRs.
+
+approvers:
+- adrcunha
+- jessiezcc
+- srinivashegde86
+- steuhs
```

Comment:
this file should have our github handles in it instead of the ones here, i.e @dlorenc @aaron-prindle @tejal29 @bobcatfish @ImJasonH